cw-orch-daemon/src/senders/cosmos.rs

```diff
@@ -132,6 +132,28 @@ impl Wallet {
         self.options.clone()
     }
 
+    /// Replaces the private key that the [CosmosSender] is using with the provided mnemonic string.
+    /// If you want more control over the derived private key, use [Self::set_private_key]
+    pub fn set_mnemonic(&mut self, mnemonic: impl Into<String>) -> Result<(), DaemonError> {
+        let secp = Secp256k1::new();
+
+        let pk = PrivateKey::from_words(
+            &secp,
+            &mnemonic.into(),
+            0,
+            self.options.hd_index.unwrap_or(0),
+            self.chain_info.network_info.coin_type,
+        )?;
+        self.set_private_key(pk);
+        Ok(())
+    }
+
+    /// Replaces the private key the sender is using
+    /// If you only need to change the mnemonic, use [Self::set_mnemonic]
```

Comment:
Changing the mnemonic = changing the private key. 
```suggestion
    /// You can use a mnemonic to overwrite the key using [Self::set_mnemonic]
```